include/z64debug.h

```diff
@@ -0,0 +1,36 @@
+#ifndef Z64DEBUG_H
+#define Z64DEBUG_H
```

Comment:
With files including `z_debug` (regs), `z_debug_display` (draw debug), and `z_debug_mode` (this file), is this worth having a more specific name than `z64debug.h` or could it contain other debug features down the road? I suppose some of these functions in `z_debug_mode` come from OoT's `z_debug` though?

Reply:
That makes sense. I decided to rename it to `z64debug_text.h` (and also do one for `z64debug_display.h`), perhaps we even add a general `z64debug.h` that includes all these other debugging systems.

Yeah these mostly come from `OoT`'s `debug`, though it adds another system for printing to the screen (though that could be the stubbed version in 'OoT' but :shrug: ).